.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Before submitting, please make sure you run a vagrant box running Aurora with the latest version shown in .auroraversion
+and run go test from the project root.
+
```

Comment:
Could you include this Link in http://aurora.apache.org/documentation/latest/getting-started/vagrant/